configure.d/config_os_functions

```diff
@@ -25,15 +25,16 @@ AC_TYPE_SIGNAL
 AC_CHECK_FUNCS([rand   random srand srandom lrand48 srand48])
 
 #  Library:
-AC_CHECK_FUNCS([asprintf        closedir        fgetc_unlocked   ] dnl
-               [flockfile       funlockfile     getipnodebyname  ] dnl
-               [gettimeofday    getlogin                         ] dnl
-               [if_nametoindex  mkstemp                          ] dnl
-               [opendir         readdir         regcomp          ] dnl
-               [setenv          setitimer       setlocale        ] dnl
-               [setsid          snprintf        strcasestr       ] dnl
-               [strdup          strerror        strncasecmp      ] dnl
-               [sysconf         times           vsnprintf        ] )
+AC_CHECK_FUNCS([asprintf         closedir         endnetgrent  ] dnl
```

Comment:
Please rework this change such that the number of changed lines is reduced.

Reply:
The order inside the library `AC_CHECK_FUNCS` is alphabetical. So either we have to change the amount to lines as above or we do not order the functions alphabetical and add one line like this:
```
+    [endnetgrent   getnetgrent  setnetgrent    ] dnl
```

Reply:
Please preserve the alphabetical order.

I think that adding a line with fewer than three function names is fine.